R/LScommon.R

```diff
@@ -1196,7 +1196,7 @@ hdi.density    <- function(object, credMass=0.95, allowSplit=FALSE, ...) {
     ))
   } else {
     return(ggplot2::scale_y_continuous(
-      ifelse (is.null(allLines), gettext("Probability"), gettext("Density")),
+      if (!is.null(yName)) yName else ifelse (is.null(allLines), gettext("Probability"), gettext("Density")),
```

Comment:
This is not ideal, how about writing a good old `if`, `else if`, `else` statement instead of mixing `if` and `ifelse()`??

Reply:
done